catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -463,4 +487,28 @@
     <service ref="featureService"
              interface="org.codice.ddf.catalog.ui.query.geofeature.FeatureService"/>
 
+    <bean id="zipSplitter" class="org.codice.ddf.catalog.ui.metacard.ZipSplitter">
```

Comment:
@pklinef I left the ZipSplitter in blueprint so people could see how splitting works. However, I could make this a metatype that must be started manually.

Reply:
I left a comment on the `ZipSplitter` class.  I could see this being moved into its own bundle.  I agree that there should be a way to disable it or have it off by default.